client/src/deploy/creation_common.rs

```diff
@@ -767,12 +766,80 @@ pub(super) fn construct_deploy(
         session,
         &secret_key,
         &mut rng,
-    );
+    )
+}
 
+pub(super) fn deploy_into_params(deploy: Deploy) -> PutDeployParams {
     let deploy = deploy
         .to_json()
         .as_object()
         .unwrap_or_else(|| panic!("should encode to JSON object type"))
         .clone();
     PutDeployParams { deploy }
 }
+
+pub(super) fn construct_deploy(
+    matches: &ArgMatches<'_>,
+    session: ExecutableDeployItem,
+) -> PutDeployParams {
+    let deploy = parse_deploy(matches, session);
+    deploy_into_params(deploy)
+}
+
+pub(super) mod output {
+    use super::*;
+
+    const ARG_NAME: &str = "output";
+    const ARG_SHORT_NAME: &str = "o";
+    const ARG_VALUE_NAME: &str = "/path/to/file";
```

Comment:
In other places (e.g. secret-key arg) we set the `ARG_VALUE_NAME` to `"PATH"`.  I think we should be consistent, including with the case.  So I'd suggest we either change all path args to show `PATH` since that works for files or folders.

It might be worth considering moving a few of these consts to common.rs to cut down a bit on the chances of inconsistencies.  Seems like candidates are `"PATH"`, `"HEX STRING"`, and `"STRING"`?

Note that the path arg in query_state.rs should remain as is - it doesn't represent a filesystem path and I'd like to leave it as is to help explain what it represents.